tests/Fixer/Operator/TernaryToElvisOperatorFixerTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Operator;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Operator\TernaryToElvisOperatorFixer
+ */
+final class TernaryToElvisOperatorFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideFixCases
+     *
+     * @param mixed      $expected
+     * @param null|mixed $input
```

Comment:
expected is always `string`
input is always `?string`

Reply:
Oh, this was filed by php-cs-fixes itself. Maybe something related to #3516?

Reply:
you mean that `@param mixed ...` were added ? yes, if there is docblock that doesn't document params, they are added with generic `mixed`. as, like here, just by `looking at code` we don't know what the types are, so fixer put generix `mixed` pseudotype.
yet, by writing the code, we do know what are the types, so we could use `string / ?string`.
putting `mixed` is some kind of reminder "maybe you shall document the types?" ;)